### PR TITLE
Remove AddressBook.framework

### PR DESCRIPTION
**Pull request by mbrandonw**
We are getting some strange start up crashes on iOS 9 devices:

```
Dyld Error Message: 
Dyld Message: Symbol not found: _kABPersonAddressCityKey 
Referenced from: /private/var/containers/Bundle/Application/0CD4EEF4-5A94-4891-9FBF-FA459B87F3F9/Kickstarter.app/Frameworks/Kickstarter_Framework.framework/Kickstarter_Framework 
Expected in: dyld shared cache 
in /private/var/containers/Bundle/Application/0CD4EEF4-5A94-4891-9FBF-FA459B87F3F9/Kickstarter.app/Frameworks/Kickstarter_Framework.framework/Kickstarter_Framework 
Dyld Version: 390.7
```

The `_kABPersonAddressCityKey` is something in AddressBook.framework, which we only link in our app because of Firebase (a dependency of live streaming) requires it. I tried just removing it to see what happens... everything still seems to work?

### CODE REVIEW

File: Kickstarter.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -158,6 +158,7 @@
 		9DDE1F701D5924AC0092D9A5 /* Checkout.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 9DDE1F6F1D5924AC0092D9A5 /* Checkout.storyboard */; };
 		9DDE1F721D5925A90092D9A5 /* CheckoutViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9DDE1F711D5925A90092D9A5 /* CheckoutViewController.swift */; };
 		9DEE3B561D1D819D0020C2BE /* ProjectActivityBackingCellViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9DEE3B241D1D81950020C2BE /* ProjectActivityBackingCellViewModel.swift */; };
+		A701B5C31EB27DD7001D4E5F /* PassKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = A701B5C21EB27DD7001D4E5F /* PassKit.framework */; settings = {ATTRIBUTES = (Weak, ); }; };
```
Comment (mbrandonw) [inline Kickstarter.xcodeproj/project.pbxproj]: people also talked about adding a optional PassKit.framework helps? Note the `{ATTRIBUTES = (Weak, );`

File: Kickstarter-iOS/Views/Controllers/LiveVideoViewController.swift [hunk 2]
```diff
@@ -173,12 +173,13 @@ public final class LiveVideoViewController: UIViewController {
   }
 
   private func addAndConfigureSubscriber(stream: OTStream) {
-    let subscriber = OTSubscriber(stream: stream, delegate: nil)
```
Comment (justinswart) [inline Kickstarter-iOS/Views/Controllers/LiveVideoViewController.swift]: This is a failable initializer in OpenTok 2.10.2

File: Makefile [hunk 3]
```diff
@@ -104,7 +104,7 @@ secrets:
 		|| true; \
 	fi
 
-OPENTOK_VERSION = 2.10.0
+OPENTOK_VERSION = 2.10.2
```
Comment (justinswart) [inline Makefile]: Bumped to latest OpenTok version.

**Comment by justinswart** [comment]
Thanks for looking into this!

So as far as I can recall these dependencies were because of OpenTok and not Firebase, but it appears that AddressBook is no longer needed in the latest OpenTok: https://tokbox.com/developer/sdks/ios/

To be sure that we're on the version that matches that list of dependencies I've also incremented our OpenTok version in the `Makefile`.

According to that list we also no longer need `CFNetwork`.

**Comment by willmanduffy** [comment]
[[ZD] Updated app on Apple IOS 9.3.2 and it immediately crashes and closes. ](https://trello.com/c/Pt1TfhNT/3134-zd-updated-app-on-apple-ios-9-3-2-and-it-immediately-crashes-and-closes)

**Pull request merged**